src/test/java/io/anserini/search/topicreader/TopicReaderTest.java

```diff
@@ -726,4 +727,57 @@ public void testBackgroundLinkingTopics() {
         "sun-erupts-to-mark-another-bastille-day-aurora-possible-in-new-england-sunday-night/",
         topics.get(topics.lastKey()).get("url"));
   }
+
+  @Test
+  public void testEpidemicQATopics() {
+    SortedMap<Integer, Map<String, String>> consumer_topics;
+    consumer_topics = TopicReader.getTopics(Topics.EPIDEMIC_QA_CONSUMER_PRELIM);
+
+    // No consumer questions from CQ035 to CQ037
+    assertEquals(42, consumer_topics.keySet().size());
+    assertEquals(1, (int) consumer_topics.firstKey());
+    assertEquals("what is the origin of COVID-19",
+                 consumer_topics.get(consumer_topics.firstKey()).get("question"));
+    assertEquals("CQ001", consumer_topics.get(consumer_topics.firstKey()).get("question_id"));
+    assertEquals("coronavirus origin", consumer_topics.get(consumer_topics.firstKey()).get("query"));
+    assertEquals("seeking information about whether the virus was designed in a lab or occured "
+                 + "naturally in animals and how it got to humans",
```

Comment:
I think our convention is `+` on previous line?

Yea, we should install a linter, I know...